### DISCUSSION TITLE
GitHub releases vs Nuget version

### DISCUSSION THREAD

**Discussion by ghost** [IonxSolutions/serilog-sinks-syslog#79] [General]
Hello,

[NuGet](https://www.nuget.org/packages/Serilog.Sinks.SyslogMessages#versions-body-tab) shows some versions that are not visible in the [releases section of this GitHub project](https://github.com/IonxSolutions/serilog-sinks-syslog/releases): 3.0.2 and 4.0.0.
Are these NuGet versions still coming from this project?
Also, 4.0.0 is a major release, so is there any API change or anything to look out for in particular? (I couldn't find any changelog short of sifting through the commits)

Thanks,
Ben

**Comment by colin-anderson** [comment]
Sorry, I forgot to create the GitHub releases for those versions 😅 - it's done now though!

Regarding the major release, it was simply because we've bumped major versions of the Serilog libraries we depend on - no changes to APIs.

Changelog: https://github.com/IonxSolutions/serilog-sinks-syslog/compare/v3.0.2...v4.0.0

**Reply by ghost** [reply]
Perfect, thanks!
